### DISCUSSION TITLE
Android TV support

### DISCUSSION THREAD

**Discussion by jdk-21** [jellyflix-app/jellyflix#183] [Ideas]
Android TV seems to be possible to implement. The app can already run on Android TV, but there are many UI improvements that have to be made:

- [ ] correct autofocus on every screen
- [ ] library screen needs filters, that can be easily navigated via dpad
- [ ] login screen should be replaced with quick connect 
- [ ] video controls are currently not working with dpad -> ExoPlayer implementation

On Android TV we should definetly use ExoPlayer. This should improve performance and should enable us to play Dolby Vision and HDR10+ content (media-kit only supports tone-mapping).

**Comment by vortex91** [comment]
any progress on this?

**Comment by jdk-21** [comment]
The new Android TV version seems to have completely broken the navigation with Flutter. This could of course be an emulator issue, but I don't have an Android TV device to test that.
The app runs on Android TV but you won't be able to navigate.

**Comment by michalss** [comment]
Any eta to make this work pls ?

**Comment by High-rider** [comment]
Any future plans for Android tv app?

**Comment by acordtag** [comment]
Hello, I could donate an AndroidTV device.
